src/interaction-handlers/ticket/categorizeCrash.ts

```diff
@@ -37,11 +37,14 @@ export class ButtonHandler extends InteractionHandler {
         );
       case "launcher":
         return channel.send(`You'll need to send your crash log. Please follow these instructions:
-1. Open your SkyClient folder. See this: https://youtu.be/BHIM2htfMk8
+1. Open your SkyClient folder. If you use the vanilla launcher, see this: https://youtu.be/BHIM2htfMk8
 2. - **If you saw a "View Crash Report" button in the launcher:**
      Go to the \`crash-reports\` folder and upload the most recent file here.
    - **If you didn't see the button:**
      Go to the \`logs\` folder and upload the file called \`latest\` or \`latest.log\`.
+   - **If you use Prism, Modrinth (app) or another non-vanilla launcher:***
```

Comment:
We only "officially" support Prism and Modrinth. This is probably better phrasing:
```suggestion
   - **If you use the Prism or Modrinth Launcher:***
```